datasets/hr/public_officials/crawler.py

```diff
@@ -72,8 +70,8 @@ def make_affiliation_entities(
         no_end_implies_current=True,
         categorisation=categorisation,
         propagate_country=True,
-        start_date=h.parse_date(start_date, DATE_FORMATS).pop(),
-        end_date=h.parse_date(end_date, DATE_FORMATS).pop(),
+        start_date=h.extract_date(context.dataset, start_date).pop(),
+        end_date=h.extract_date(context.dataset, end_date).pop(),
```

Comment:
might be solved if we move extract_date into make_occupancy

Reply:
what do you mean? it's already inside `make.occupancy`, isn't it?

Reply:
we've made this change now.
in this comment's selection, we're calling extract_date _in the call to_ make_occupany. After the changes now in this PR, we're calling extract_date (via apply_date) _within_ make_occupancy